lib/exercises.js

```diff
@@ -1,16 +1,35 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
-function grouped_anagrams(strings) {
-  throw new Error("Method hasn't been implemented yet!");
+// Time Complexity: O(n) where n is the length of the input array
```

Comment:
Since you're sorting the strings this would be O(n * m log m) where `m` is the average length of a string.  If  you can assume the strings are small, you can drop the m term.